test/main.js

```diff
@@ -128,3 +146,27 @@ describe('Offline mode feature tests after a server restart', function() {
 
     after(function () { return stopRestbase(); });
 });
+
+describe('Log', function () {
+
+    var assert = require('./utils/assert.js');
+
+    this.timeout(20000);
+
+
+    it('should only have PUT, GET, and POST request entries', function() {
```

Comment:
Should we check this after each test? Having a global check for log entries sounds like it would often fail / need to be updated.